index.html

```diff
@@ -5,57 +5,208 @@
     <link rel="stylesheet" href="style.css">
-    <title>ERNEST Portfolio Website</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>MiddlePart Mobile Website</title>
 </head>
 <body>
-    <!-- defining the header and wraping it in divs and will contain the logo -->
-    <div class="main_container">
-        <div class="inside_container">
-            <header class="main_header">
-              <nav class="navbar">
-                <a href="#" class="my-portfolio-logo">
-                    <img src="./assets/MyLogo.png" alt="My logo"/>
-                </a>
-                <a>
-                    <img src="./assets/menu.png" alt="menu-icon"/>
-                </a>
-              </nav>
-            </header>
-    <!-- Defining the main section for the description of the developer -->
-            <main class="main_section">
-                <div class="container">
-                   <div class="welcome_note"> 
-                       <h1 class="h1">I am Ernest Lasten. Glad to see you!</h1>
-                    </div>
-                        <p class="main_content">
-                            I’m a software developer! I can help you build a product ,
-                            feature or website Look through some of my work and experience!
-                            If you like what you see and have a project you need coded, 
-                            don’t hestiate to contact me.
-                        </p>
-                    <div class="connect_links">
-                        <a href="#" class="connect_text">LET'S CONNECT</a>                    
-                        <ul class="links_list">
-                            <li class="media_links">
-                            <a href="#"><img src="./assets/twitterIcon.svg" alt="icon for twitter"/></a>
-                            </li>
-                            <li class="media_links">
-                            <a href="#"><img src="./assets/linkedInIcon.png" alt="icon for linkedIn"/></a>
-                            </li>
-                            <li class="media_links">
-                            <a href="#"><img src="./assets/mIcon.svg" alt="icon for Medium"/></a>
-                            </li>
-                            <li class="media_links">
-                            <a href="#"><img src="./assets/gitHubIcon.svg" alt="icon for GitHub"/></a>
-                            </li>
-                            <li class="media_links">
-                            <a href="#"><img src="./assets/peaceIcon.svg" alt="icon for M"/></a>
-                            </li>
-                        </ul>
-                    </div>
-                </div>
-            </main>
+  <div class="main_container"> 
+    <div class="inside_container"> 
+      <section class="snapshots">
+        <div class="title-grid">
```

Comment:
- [ ] Nice working on your project section, kindly re-align the cards so they match how it looks in the Figma Design you can view the below screenshot
- [ ] Kindly add a white `background` for the card, so it aligns with what you have in the Figma Template.

## Your Design
![cc-1](https://user-images.githubusercontent.com/48371156/166123245-b5555e1a-c9e3-4b9d-9434-0a67c10cb32a.png)

## Figma Template
![cc-2](https://user-images.githubusercontent.com/48371156/166123246-2cf8ae61-a7a6-41c7-8749-0895c1d60881.png)